tests/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.15-alpine AS builder
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux
+
+WORKDIR /build
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN cd e2e && go test -c -o e2e-tests 
+
+FROM alpine:3.13
+
+RUN apk update && apk add --no-cache --upgrade curl python3 py3-pip
+RUN pip install awscli
```

Comment:
this is awscli v1, while we use v2 on our laptops and I think all the new niceties will be in v2
There is no Alpine package for it, but https://github.com/aws/aws-cli/issues/4685#issuecomment-829600284 hints at a way to get it built and keep the image small

Reply:
Regarding this, it worked!!!  Thanks for that. The image went down from 96MB to 76MB, but the amount of code added within the Dockefile doesn't feel worth :-(